### PR TITLE
Add 'Building from tarball' to README.md

### PR DESCRIPTION
**Pull request by rockdaboot**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -10,24 +10,15 @@ cc = meson.get_compiler('c')
...
+# We need to know the build type to determine what .lib files we need on Visual Studio
+# for dependencies that don't normally come with pkg-config files for Visual Studio builds
+buildtype = get_option('buildtype')
 
 notfound = dependency('', required : false)
 libidn2_dep = notfound
 libicu_dep = notfound
 libidn_dep = notfound
 libunistring = notfound
-libiconv = notfound
```
Comment (tp-m) [inline meson.build:30]: Is this being removed again on purpose? This was just added in a previous PR #136 

Or was it removed by accident during conflict resolution?

File: meson.build [hunk 2]
```diff
@@ -95,7 +86,6 @@ endif
 if libidn2_dep.found() or libidn_dep.found()
   # Check for libunistring, we need it for psl_str_to_utf8lower()
   libunistring = cc.find_library('unistring')
-  libiconv = cc.find_library('libiconv')
```
Comment (tp-m) [inline meson.build:98]: Is this being removed again on purpose? This was just added in a previous PR #136 

Or was it removed by accident during conflict resolution?

**Review by tp-m — COMMENTED** [review]
Some meson bits got removed here which seems to break the build: 
```
subprojects\libpsl\src\meson.build:19:0: ERROR: Unknown variable "libiconv".
```

File: meson.build (hunk 2 above)
Comment (rockdaboot) [reply meson.build:98]: The meson.build seems to be auto-generated from time to time, not sure if by 'make' or './configure'. Unnoticed by me this time. Sorry for that, but if that ever happens again, I have to remove `meson.build` from the repo (auto-generated files don't belong there).

File: meson.build (hunk 1 above)
Comment (rockdaboot) [reply meson.build:30]: See comment below.

**Pull request merged**